citation/models.py

```diff
@@ -697,6 +708,31 @@ def __str__(self):
                                                              container=self.container)
 
 
+class URLStatusLogs(models.Model):
+    PLATFORM_TYPES = Choices((PlatformTypes.COMSES, _('CoMSES')),
+                     (PlatformTypes.OPEN_SOURCE, _('Open Source')),
+                     (PlatformTypes.PLATFORM, _('Platform')),
+                     (PlatformTypes.JOURNAL, _('Journal')),
+                     (PlatformTypes.PERSONAL, _('Personal')),
+                     (PlatformTypes.INVALID, _('Invalid')),
+                     (PlatformTypes.OTHERS, _('Others'))
+                     )
+    pub_id = models.ForeignKey(Publication, related_name='url_status', null=True, blank=True, db_constraint=False)
+    url = models.URLField(blank=True, max_length=500)
+    date_added = models.DateTimeField(auto_now_add=True,
+                                      help_text=_('Date this url was last verified'))
+    date_modified = models.DateTimeField(auto_now=True,
+                                         help_text=_('Date this url status was last modified on this system'))
+    text = models.TextField(blank=True, help_text=_('contains information about the url header'))
```

Comment:
perhaps this should just be named `headers` and set to whatever headers the requests library gave us in the response?